CHANGELOG.md

```diff
@@ -24,6 +24,8 @@ Line wrap the file at 100 chars.                                              Th
 
 ## [Unreleased]
 ### Added
+
+- Add fish shell completions for the mullvad CLI.
```

Comment:
This should be under a `#### Linux` header since it's platform specific.

Reply:
My bad! It's for two platforms. So keeping it under the generic header is fine. It could be extended to say "on Linux and macOS" but that's just nitpick. I don't think fish exists for Windows anyway? Resolved.